hack/tools/install_kubebuilder.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
I am not sure having a folder with duplicate name is a best practice, since there is already `tools` folder in root of the repository.

Reply:
Yes sure, removing duplicate folder.